kratos/processes/calculate_distance_to_path_process.h

```diff
@@ -0,0 +1,404 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+#pragma once
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "processes/process.h"
+#include "containers/model.h"
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+/**
+ * @brief How the distance is computed enum
+ */
+enum class DistanceComputed
+{
+    ERROR,
+    NO_RADIUS,
+    RADIUS_PROJECTED,
+    RADIUS_NOT_PROJECTED_OUTSIDE,
+    RADIUS_NOT_PROJECTED_INSIDE
+};
+
```

Comment:
Same comment as for the auxiliary PR you did for this. I don't like the idea of having these global enum definitions. Even worse if the unique purpose is for the `static` method you are creating inside the class.